config/mvi_schema/mock_mvi_responses.yml.example

```diff
@@ -84,9 +84,20 @@ find_candidate:
     given_names: ['JERRY', 'M']
     icn: '1008709435V263289^NI^200M^USVHA^P'
     mhv_id: '12210827^PI^200MH^USVHA^A'
-    vba_corp_id: ~
+    vba_corp_id: '600036156'
```

Comment:
I didn't get the information after the carets, and it might be better to just omit it than to fake it.